GetValueSet.js

```diff
@@ -1,10 +1,17 @@
 function loadValueSet(data) {
   const valueSetMap = [];
```

Comment:
NIT: I think based on the way we're using this variable, it would make the most sense to initialize it as a javascript `Object`, rather than an `Array`. You can do that by swapping the square brackets (`[]`) for curly braces (`{}`).